fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/RecordQueryPlannerConfiguration.java

```diff
@@ -37,19 +37,25 @@
     private final boolean checkForDuplicateConditions;
     private final boolean deferFetchAfterUnionAndIntersection;
     private final boolean optimizeForIndexFilters;
+    private final int maxTaskQueueSize;
```

Comment:
Can you add a comment somewhere that these parameters being `0` (which is their default) indicate that we don't enforce the limit?

Reply:
Done.